scenes/startup.txt

```diff
@@ -90,6 +92,17 @@
 *create test_stat_charisma 0
 *create test_stat_wisdom 0
 
+*comment TEST DATA for OBJECTS
+*create testobj_fieldname_1 "testfield1"
```

Comment:
What do you think about having just "field" instead of "filedname"?

Reply:
My thinking was that these are not the fields, in that the sense that you would never access them directly.
However, I'm quite happy to change it. Just let me know what you think is best.

Reply:
I think field is clear enough... I certainly don't think it's something that would result in repeat mistakes, and it's a big win in terms of conciseness to have 'field'. I'd recommend the change if you don't object to it.

Reply:
+1 to just field

Reply:
The votes are all going to 'field', so I'll change it to that.

Reply:
Done